client/test/scenario.js

```diff
@@ -250,7 +251,7 @@ Scenario.prototype.withWorkflows = function withWorkflows(
   })}`;
 
   const response = Array.isArray(workflows)
-    ? { executions: workflows }
+    ? { executions: workflows, nextPageToken: '' }
     : workflows;
 
   this.api.getOnce(url, response);
```

Comment:
nit: so it does just one api call to get the page of All workflows now?

Reply:
It makes 2 API calls (open & closed) at the same time to populate the All workflows list.

Reply:
This will test the exact URL is only called once. 
i.e:
/workflows/open
/workflows/closed

are considered unique and will assert they are only called once each.

Reply:
yep, i see you do it in parallel above:

```
.withWorkflows({ status: 'open' })
.withWorkflows({ status: 'closed', startTimeOffset: 30 });
```